AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSCosmosDbIngestion.h

```diff
@@ -6,7 +6,18 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Error domain for Storage.
+ */
+static NSString *const MSDataStorageErrorDomain = @"MSDataStorageErrorDomain";
```

Comment:
It depends on what we expose to customers.
Current implementation only expose error code to customers so it could be in the header but it might be changed soon. Related to [this](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1436/files#r269817126)

Reply:
Waiting for final decision.